.github/workflows/build.yml

```diff
@@ -215,6 +215,11 @@ jobs:
         run: |
           env
           cargo build --release --all
+      - name: Build Tracing Node
+        if: contains(github.event.pull_request.labels.*.name, 'A10-evmtracing')
+        run: |
+          env
+          cargo build --release -p moonbeam --features evm-tracing
```

Comment:
This is going to override the previous built binary ? 
So the normal tests won't run or the normal binary but on the evm-tracing one, right ?
Is that a problem ?

Reply:
> This is going to override the previous built binary ?

Yes, it overwrites the previous binary. The goal is to make sure it still compiles without tracing, but to play the tests with tracing enabled.

> So the normal tests won't run or the normal binary but on the evm-tracing one, right ?

Yes, absolutely, and this is intentional, to ensure that tracing nodes work at least as well as regular nodes.